### PR TITLE
[13.0][FIX] account_financial_report: Set the correct value of column 'Cumul cur.' in general ledger

### PR DESCRIPTION
**Pull request by victoralmau**
Changes done:
- [x]  Improve domain to reduce code in general ledger.
- [x] Set the correct value of column 'Cumul cur.' in general ledger.
- [x] Change `partner_domain` to `misc_grouped_domain` in general_ledger.

Please @pedrobaeza and @sergio-teruel can you review it?

@Tecnativa TT39754

### CODE REVIEW

File: account_financial_report/report/templates/general_ledger.xml [hunk 1]
```diff
@@ -352,7 +352,41 @@
...
+                                    t-set="domain"
+                                    t-value="[('account_id', '=', account['id']),
+                                             ('date', '&lt;', o.date_from)]"
+                                />
+                                <span
+                                    t-att-domain="domain"
+                                    res-model="account.move.line"
+                                >
+                                    <t
+                                        t-raw="account_or_group_item_object['init_bal']['bal_curr']"
```
Comment (sergio-teruel) [inline account_financial_report/report/templates/general_ledger.xml:341]: Are you sure of use t-raw?

Comment (victoralmau) [reply account_financial_report/report/templates/general_ledger.xml:341]: Yes, it is exactly the same code as the previous cell and both cells show the correct value.

**Review by sergio-teruel — APPROVED** [review]

File: account_financial_report/report/templates/general_ledger.xml [hunk 2]
```diff
@@ -362,6 +396,12 @@
             </div>
             <!-- Display each lines -->
             <t t-set="total_bal_curr" t-value="0" />
+            <t t-if="account_or_group_item_object['init_bal']['bal_curr']">
+                <t
+                    t-set="total_bal_curr"
+                    t-value="account_or_group_item_object['init_bal']['bal_curr']"
+                />
+            </t>
```
Comment (pedrobaeza) [inline account_financial_report/report/templates/general_ledger.xml]:
```suggestion
            <t t-set="total_bal_curr" t-value="account_or_group_item_object['init_bal']['bal_curr'] or 0" />
```

File: account_financial_report/report/templates/general_ledger.xml [hunk 3]
```diff
@@ -352,7 +352,41 @@
...
+                                        t-raw="account_or_group_item_object['init_bal']['bal_curr']"
+                                        t-options="{'widget': 'monetary', 'display_currency': account['currency_id']}"
+                                    />
+                                </span>
+                            </t>
+                            <t t-if="type == 'grouped_type'">
+                                <t
+                                    t-set="domain"
+                                    t-value="[('account_id', '=', account['id']),
+                                             ('partner_id', '=', group_item['id']),
```
Comment (pedrobaeza) [inline account_financial_report/report/templates/general_ledger.xml]: Although not of this PR, you can also group by tax, so this is not correct.

Comment (pedrobaeza) [reply account_financial_report/report/templates/general_ledger.xml]: And why not sharing certain parts of the domain for the whole line?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I see no values in PDF:

Selección_015

And I suspect there's nothing done on XLS part.

File: account_financial_report/report/templates/general_ledger.xml (hunk 3 above)
Comment (victoralmau) [reply account_financial_report/report/templates/general_ledger.xml]: I have added an initial commit to reduce code and re-use domain.

**Comment by victoralmau** [comment]
The changes in XLS option are already applied to show the correct total.

File: account_financial_report/report/templates/general_ledger.xml (hunk 3 above)
Comment (victoralmau) [reply account_financial_report/report/templates/general_ledger.xml]: Now `partner_id` domain is now only added when groupg by partners.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**